src/figuresClasses.ts

```diff
@@ -1,11 +1,94 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
```

Comment:
```suggestion
type Color = 'red' | 'green' | 'blue';
type Shape = 'triangle' | 'circle' | 'rectangle';

export interface Figure {
  shape: Shape;
  color: Color;
```
we can specify types more precisely

Reply:
<img width="636" alt="image" src="https://github.com/user-attachments/assets/0b6b359c-f150-4673-a8fb-8163974c84fa">
still not fixed